### DISCUSSION TITLE
JSON 文件

### DISCUSSION THREAD

**Discussion by Burninghamster** [illacloud/illa-builder#763] [General]
您好，是否有提供导出应用为 JSON 文件的功能，如果没有后续会考虑上线此功能吗？

**Comment by wenzhuo2022** [comment]
您好，短期内不会支持导出JSON，再往后功能更完善的时候会做支持。可以了解一下您会在什么情况下使用导出JSON文件的功能吗？

**Reply by Burninghamster** [reply]
像我们这种偏传统企业，不是互联网公司，大部分场景是后台管理比较多，导出json是为了方便管理维护多个项目，官方也可以出json模板方便其他用户学习功能，社区资料比较少，很多功能都不知道怎么用的，比如像码匠实用的就导出json，快速搭建后台管理界面，可扩展性，然后还有权限。像适配啥的都是可有可无的

**Reply by wenzhuo2022** [reply]
收到，这个会加入我们的计划里～
